### DISCUSSION TITLE
Clicking into fields to edit (or icon to edit field) not available with Documents

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#15650] [How to]
Why does the `Document `screen not allow to edit fields by licking in the field or on an edit icon next to the field as it is possible with other modules?

![Screenshot 2022-01-21 122421](https://user-images.githubusercontent.com/10230144/150519230-649582e5-0053-4f2a-bf26-84e30928adaa.png)

versus

![example](https://user-images.githubusercontent.com/10230144/150519366-13a9d831-d029-43eb-8345-2ea91525fb22.png)

**Comment by s2b-git** [comment]
Inheritance from vtiger 6. Remember it was fixed. Little java thing, Honestly, Document module could have lots of improvement. EDI becomes important part of business processes here and there :-)
